### ISSUE TITLE
Crash when crafting with the auto put in item feature in JEI (SimpleStorageNetwork)

### ISSUE THREAD

**Issue by wiviw5** [ftog-thorgal/Unbound#1]
# Version of Pack: 0.6 Beta
Issue with the mod: Simple Storage Network
Added mods to pack: Schematica + LunatriusCore
Done in singleplayer test world
_____________________________________________________________________________________________________________
Steps to Reproduce:
1: Put down a storage setup consisting of a iron chest connected with the with the necessary parts aswell
(To be Specific a storage Request Table, Storage Network Master, Cables Connecting them,A storage link able connecting it to the Iron Chest)
2: Put in something such as planks into the storage system and some other random item (Not sure if the other items had something to do with it)
3: Go into JEI and go to the crafting table recipe and press the plus button that puts stuff into the grid
4: Crash
_____________________________________________________________________________________________________________
In summary this is most likely a crash to do with either JEI not being able to put items into the slots or Simple storage networks. So just put a reminder for this bug for everyone else. Also minecraft "froze" and didn't auto close itself.
_____________________________________________________________________________________________________________
(Updates) Only happens if the item that is being crafted has the necessary materials to craft it. If it does not or is not a crafting recipe or furnace recipe then it does not craft. 
V.2 Tried without schematica and LunatriusCore and only boots to menu.
V.3 It crashed without them again its pretty random

**Comment by ftog-thorgal** [comment]
This issue has been reported to the mod author here: https://github.com/PrinceOfAmber/Storage-Network/issues/38
Hopefully it will be fixed soon.

**Comment by ftog-thorgal** [comment]
This issue has been fixed in the latest version.
